Profiles/BCU/responses.yml

```diff
@@ -101,6 +101,11 @@ Chat:
     madsteve79: Mad Steve 79
     20thCenturyDinosaur: 20th Century Dinosaur
     Kefka40: Kefka 40
+    PyroHeromatt: Pyro Hero Matt
+    DrMutagen: Doctor Mutagen
+    Nightbot: Night bot
```

Comment:
I'm gonna miss "Night Butt"

Reply:
We need to allow these to be randomized or something. lol